x-pack/plugin/esql/qa/testFixtures/src/main/resources/string.csv-spec

```diff
@@ -2796,3 +2748,14 @@ book_no:keyword | author_encoded:keyword | title_encoded:keyword
 1463            | J.%20R.%20R.%20Tolkien | Realms%20of%20Tolkien%3A%20Images%20of%20Middle-earth
 ;
 
+ascii
+required_capability: ascii
+// tag::ascii[]
+ROW a = "Hello\n\t 世界! 🌍 Café naïve résumé こんにちは 🎉 中文测试 αβγδε 日本語テスト 🚀🔥💧🪨" | EVAL x = ASCII(a) | KEEP x;
+// end::ascii[]
```

Comment:
Because this is going to be displayed in the docs it might be nicer to format it like:
```
ROW s = [
  "Hello world!\n\t",
  "世界!",
  "🌍",
  "Café naïve résumé",
  "こんにちは",
  " 🎉",
  "中文测试",
  "αβγδε",
  "日本語テスト"
  "🚀🔥💧🪨"
]
| EVAL s = TO_ASCII(s)
```

Otherwise it gets really wide to fit into the screen.